src/Visitor.php

```diff
@@ -151,7 +151,7 @@ public function httpHeaders() : array
      */
     public function userAgent() : string
     {
-        return $this->request->userAgent() ?? '';
+        return $this->request->userAgent() ?? 'Unknown User Agent';
```

Comment:
lets just return empty string instead of this. when there is no data, we should not provide a data! then how to check if the user agent exists ? it should be done like ` x == 'Unknown User Agent'` which is not idiomatic.